src/components/editor/property/InputLiteral.jsx

```diff
@@ -64,15 +55,7 @@ export class InputLiteral extends Component {
     this.setState({ content_add: userInput })
   }
 
-  notRepeatableAfterUserInput = (userInputArray, currentcontent) => {
-    let newId = null
-
-    if (this.props.formData === undefined || this.props.formData.items < 1) {
-      newId = this.addUserInput(userInputArray, currentcontent)
-      this.setState({ disabled: true })
-    }
-    return newId
-  }
+  notRepeatableAfterUserInput = (userInputArray, currentcontent) => this.addUserInput(userInputArray, currentcontent)
```

Comment:
Is this necessary anymore? If it just calls `addUserInput()` then can line 78 just call `addUserInput()` regardless of whether the field is repeatable?

Reply:
Thanks. That's a good suggestion. I've made that change.